### ISSUE TITLE
[BUG] Missing dir && file for backup

### ISSUE THREAD

**Issue by psstoyanov** [retro-center/rcade_releases#6]
### Description
With the first update to `1.0.2` from `1.0.1` , the system strongly encourages the user to backup the system.

However, the backup can't be created.

### Details

Looking at the logs:

- When initiating the backup, the user is presented with the following message
![20220105_221814_HDR](https://user-images.githubusercontent.com/3979134/148299161-615bc8e7-23c6-4278-97a6-11cd2e1e2136.jpg)

The actual report from uart indicates that the expected directory and archive file are not present:
<details>
<summary> missing /rcade/share/.emulationstation/backups/rootfs_backup.tar.xz </summary>

```
tar: Removing leading `/' from member names
tar (child): /rcade/share/.emulationstation/backups/rootfs_backup.tar.xz: Cannot open: No such file or directory
tar (child): Error is not recoverable: exiting now
tar: /rcade/share/.emulationstation/backups/rootfs_backup.tar.xz: Cannot write: Broken pipe
tar: Child returned status 2
tar: Error is not recoverable: exiting now
```
</details>

- Manually creating the directory and an empty file with the expected name allows the process to proceed
![20220105_222748_HDR](https://user-images.githubusercontent.com/3979134/148298817-f51a7581-b9d4-4b9a-8476-822b8690a45c.jpg)
<details>
<summary> created missing dir and file allow for the process to proceed </summary>

```
# mkdir backups
# cd !$
cd backups
# pwd
/rcade/share/.emulationstation/backups
# touch rootfs_backup.tar.xz 
# tar: Removing leading `/' from member names    # Indicating that the process has begun
```

</details>

**Comment by retro-center** [comment]
Running this over SSH before starting the back up will resolve this, and a fix for this will be included in the next release :) Thank you for reporting this!

   mkdir -p /rcade/share/.emulationstation/backups

**Comment by retro-center** [comment]
Resolved in 1.0.3
